api/python/arrus/utils/imaging.py

```diff
@@ -38,10 +38,11 @@ def is_package_available(package_name):
     import cupy
     import re
 
-    if not re.match("^\\d+\\.\\d+\\.\\d+$", cupy.__version__):
+    if not re.match("^\\d+\\.\\d+\\.\\d+[a-z]?\\d?$", cupy.__version__):
```

Comment:
This will not work for example for `cupy==12.0.0rc1` (https://pypi.org/project/cupy/12.0.0rc1/)